bot/exts/fun/anagram.py

```diff
@@ -0,0 +1,97 @@
+import asyncio
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+log = logging.getLogger(__name__)
+
+TIME_LIMIT = 60
+
+# Issue with the json file , gotta try it on local environment first
+ANAGRAMS_ALL = json.loads(Path("bot/resources/fun/anagram.json").read_text("utf8"))
+
+
+class Anagram(commands.Cog):
+    """Cog for the Anagram game command."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.winners = set()
+        self.correct = ""
+        self.current_channel = None
+
+    @commands.command(name="anagram", aliases=("anag", "gram", "ag"))
+    async def anagram_command(self, ctx: commands.Context) -> None:
+        """
+        Given shuffled letters which can be rearranged to form Anagrams.
+
+        If this works correctly will update this more in detail
+        """
+        if self.current_channel:
+            await ctx.send(f"An anagram is already being solved in {self.current_channel.mention}!")
+            return
+
+        # Don't let users start in a DM
+        if not ctx.guild:
+            await ctx.send(
+                embed=discord.Embed(
+                    title=random.choice(NEGATIVE_REPLIES),
+                    description="You can't start riddles in DMs",
+                    colour=discord.Colour.red()
+                )
+            )
+            return
+
+        self.current_channel = ctx.channel
+
+        anagram_letters, anagram_answers = random.choice(list(ANAGRAMS_ALL.items()))
+        scrambled_letters = anagram_letters
+        self.correct = anagram_answers
```

Comment:
```suggestion
        scrambled_letters, self.correct = random.choice(list(ANAGRAMS_ALL.items()))
```
Could avoid the re-assignment and just do it directly right?

Reply:
Made these variable changes, had to update some variables at the end of script so not committing your suggestions directly